app/schemas/images.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import BaseModel, constr
+
+URL_SAFE_REGEX = r"^[a-zA-Z0-9_-]*$"
```

Comment:
pretty strict on what you can pass in as a query. url safe, file path safe--and generally limited / kept as simple as possible.